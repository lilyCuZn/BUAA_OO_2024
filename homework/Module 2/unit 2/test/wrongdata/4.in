[1.4]21-FROM-4-TO-9
[4.0]15-FROM-4-TO-3
[5.5]12-FROM-8-TO-9
[6.2]59-FROM-4-TO-6
[8.3]6-FROM-3-TO-5
[9.3]RESET-Elevator-3-8-0.6
[11.3]35-FROM-3-TO-11
[11.7]20-FROM-5-TO-1
[11.9]32-FROM-4-TO-7
[12.7]19-FROM-9-TO-6
[13.2]RESET-Elevator-2-8-0.5
[14.2]22-FROM-6-TO-7
[15.5]4-FROM-9-TO-4
[15.6]62-FROM-2-TO-4
[16.1]34-FROM-4-TO-9
[16.2]69-FROM-8-TO-4
[17.7]60-FROM-8-TO-1
[18.1]2-FROM-6-TO-2
[19.1]5-FROM-2-TO-10
[20.2]9-FROM-6-TO-5
[21.2]52-FROM-5-TO-3
[23.6]48-FROM-10-TO-8
[23.7]49-FROM-1-TO-8
[24.5]55-FROM-9-TO-8
[25.9]14-FROM-4-TO-5
[26.0]8-FROM-4-TO-1
[26.2]51-FROM-4-TO-1
[26.6]29-FROM-2-TO-6
[28.2]38-FROM-2-TO-3
[28.7]31-FROM-1-TO-10
[29.7]24-FROM-3-TO-10
[31.8]23-FROM-9-TO-3
[31.9]39-FROM-9-TO-8
[32.0]64-FROM-3-TO-1
[32.2]7-FROM-7-TO-5
[33.0]RESET-Elevator-3-8-0.3
[33.4]68-FROM-9-TO-7
[34.2]RESET-Elevator-5-3-0.6
[36.2]3-FROM-6-TO-10
[36.6]61-FROM-1-TO-11
[37.0]54-FROM-4-TO-5
[37.9]27-FROM-7-TO-6
[38.1]RESET-Elevator-3-4-0.2
[39.7]37-FROM-9-TO-4
[39.9]66-FROM-9-TO-10
[40.1]41-FROM-5-TO-4
[40.5]RESET-Elevator-4-3-0.6
[40.5]42-FROM-3-TO-7
[41.3]40-FROM-4-TO-5
[41.8]13-FROM-4-TO-9
[41.9]53-FROM-7-TO-9
[42.0]50-FROM-5-TO-1
[42.6]36-FROM-7-TO-10
[43.0]16-FROM-5-TO-2
[43.0]63-FROM-2-TO-7
[43.1]70-FROM-9-TO-10
[45.0]RESET-Elevator-6-7-0.3
[45.6]RESET-Elevator-4-8-0.3
[46.6]45-FROM-11-TO-8
[46.9]10-FROM-8-TO-6
[46.9]11-FROM-1-TO-3
[47.5]65-FROM-9-TO-2
[47.6]28-FROM-3-TO-2
[48.5]RESET-Elevator-1-3-0.2
[48.6]46-FROM-3-TO-9
[48.6]43-FROM-2-TO-1
