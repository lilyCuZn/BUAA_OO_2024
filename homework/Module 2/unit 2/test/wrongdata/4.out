[   1.4850]RECEIVE-21-1
[   1.8850]ARRIVE-2-1
[   2.2850]ARRIVE-3-1
[   2.6860]ARRIVE-4-1
[   2.6860]OPEN-4-1
[   2.8870]IN-21-4-1
[   3.0870]CLOSE-4-1
[   3.4870]ARRIVE-5-1
[   3.8880]ARRIVE-6-1
[   4.0830]RECEIVE-15-2
[   4.2880]ARRIVE-7-1
[   4.4830]ARRIVE-2-2
[   4.6890]ARRIVE-8-1
[   4.8830]ARRIVE-3-2
[   5.0890]ARRIVE-9-1
[   5.0900]OPEN-9-1
[   5.2840]ARRIVE-4-2
[   5.2840]OPEN-4-2
[   5.2900]OUT-21-9-1
[   5.4840]IN-15-4-2
[   5.4900]CLOSE-9-1
[   5.5800]RECEIVE-12-1
[   5.6850]CLOSE-4-2
[   5.9810]ARRIVE-8-1
[   5.9810]OPEN-8-1
[   6.0850]ARRIVE-3-2
[   6.0860]OPEN-3-2
[   6.1820]IN-12-8-1
[   6.2800]RECEIVE-59-2
[   6.2860]OUT-15-3-2
[   6.3820]CLOSE-8-1
[   6.4860]CLOSE-3-2
[   6.7820]ARRIVE-9-1
[   6.7830]OPEN-9-1
[   6.8870]ARRIVE-4-2
[   6.8870]OPEN-4-2
[   6.9830]OUT-12-9-1
[   7.0870]IN-59-4-2
[   7.1830]CLOSE-9-1
[   7.2880]CLOSE-4-2
[   7.6880]ARRIVE-5-2
[   8.0880]ARRIVE-6-2
[   8.0890]OPEN-6-2
[   8.2890]OUT-59-6-2
[   8.3830]RECEIVE-6-3
[   8.4890]CLOSE-6-2
[   8.7830]ARRIVE-2-3
[   9.1830]ARRIVE-3-3
[   9.1840]OPEN-3-3
[   9.3790]RESET_ACCEPT-3-8-0.6
[   9.3840]IN-6-3-3
[   9.5840]CLOSE-3-3
[   9.5840]OPEN-3-3
[   9.7850]OUT-6-3-3
[   9.7870]RECEIVE-6-4
[   9.9850]CLOSE-3-3
[   9.9850]RESET_BEGIN-3
[  10.1870]ARRIVE-2-4
[  10.5880]ARRIVE-3-4
[  10.5880]OPEN-3-4
[  10.7880]IN-6-3-4
[  10.9880]CLOSE-3-4
[  11.1860]RESET_END-3
[  11.3890]ARRIVE-4-4
[  11.4180]RECEIVE-35-5
[  11.7890]ARRIVE-5-4
[  11.7890]OPEN-5-4
[  11.8190]ARRIVE-2-5
[  11.8290]RECEIVE-20-2
[  11.9900]OUT-6-5-4
[  12.0260]RECEIVE-32-4
[  12.1900]CLOSE-5-4
[  12.2190]ARRIVE-3-5
[  12.2190]OPEN-3-5
[  12.2290]ARRIVE-5-2
[  12.2290]OPEN-5-2
[  12.4190]IN-35-3-5
[  12.4290]IN-20-5-2
[  12.5900]ARRIVE-4-4
[  12.5910]OPEN-4-4
[  12.6200]CLOSE-3-5
[  12.6300]CLOSE-5-2
[  12.7910]IN-32-4-4
[  12.7960]RECEIVE-19-1
[  12.7960]OPEN-9-1
[  12.9910]CLOSE-4-4
[  12.9960]IN-19-9-1
[  13.0200]ARRIVE-4-5
[  13.0300]ARRIVE-4-2
[  13.1970]CLOSE-9-1
[  13.2790]RESET_ACCEPT-2-8-0.5
[  13.3920]ARRIVE-5-4
[  13.4200]ARRIVE-5-5
[  13.4300]ARRIVE-3-2
[  13.4300]OPEN-3-2
[  13.5970]ARRIVE-8-1
[  13.6310]OUT-20-3-2
[  13.6410]RECEIVE-20-3
[  13.6420]OPEN-3-3
[  13.7920]ARRIVE-6-4
[  13.8210]ARRIVE-6-5
[  13.8310]CLOSE-3-2
[  13.8310]RESET_BEGIN-2
[  13.8420]IN-20-3-3
[  13.9970]ARRIVE-7-1
[  14.0420]CLOSE-3-3
[  14.1920]ARRIVE-7-4
[  14.1920]OPEN-7-4
[  14.2210]ARRIVE-7-5
[  14.3060]RECEIVE-22-1
[  14.3930]OUT-32-7-4
[  14.3980]ARRIVE-6-1
[  14.3980]OPEN-6-1
[  14.5930]CLOSE-7-4
[  14.5980]OUT-19-6-1
[  14.6210]ARRIVE-8-5
[  14.6430]ARRIVE-2-3
[  14.7980]CLOSE-6-1
[  14.7990]OPEN-6-1
[  14.9990]IN-22-6-1
[  15.0220]ARRIVE-9-5
[  15.0310]RESET_END-2
[  15.1990]CLOSE-6-1
[  15.2430]ARRIVE-1-3
[  15.2430]OPEN-1-3
[  15.4260]ARRIVE-10-5
[  15.4430]OUT-20-1-3
[  15.6000]ARRIVE-7-1
[  15.6000]OPEN-7-1
[  15.6270]RECEIVE-4-4
[  15.6440]CLOSE-1-3
[  15.6870]RECEIVE-62-6
[  15.8050]OUT-22-7-1
[  15.8260]ARRIVE-11-5
[  15.8260]OPEN-11-5
[  16.0050]CLOSE-7-1
[  16.0260]OUT-35-11-5
[  16.0270]ARRIVE-8-4
[  16.0870]ARRIVE-2-6
[  16.0880]OPEN-2-6
[  16.2240]RECEIVE-34-2
[  16.2270]CLOSE-11-5
[  16.2880]IN-62-2-6
[  16.3200]RECEIVE-69-1
[  16.4270]ARRIVE-9-4
[  16.4270]OPEN-9-4
[  16.4880]CLOSE-2-6
[  16.6270]IN-4-9-4
[  16.7200]ARRIVE-8-1
[  16.7200]OPEN-8-1
[  16.7240]ARRIVE-4-2
[  16.7240]OPEN-4-2
[  16.8280]CLOSE-9-4
[  16.8890]ARRIVE-3-6
[  16.9210]IN-69-8-1
[  16.9250]IN-34-4-2
[  17.1210]CLOSE-8-1
[  17.1250]CLOSE-4-2
[  17.2280]ARRIVE-8-4
[  17.2890]ARRIVE-4-6
[  17.2890]OPEN-4-6
[  17.4890]OUT-62-4-6
[  17.5210]ARRIVE-7-1
[  17.6250]ARRIVE-5-2
[  17.6280]ARRIVE-7-4
[  17.6900]CLOSE-4-6
[  17.7800]RECEIVE-60-5
[  17.9220]ARRIVE-6-1
[  18.0280]ARRIVE-6-4
[  18.1250]ARRIVE-6-2
[  18.1810]ARRIVE-10-5
[  18.1940]RECEIVE-2-1
[  18.3230]ARRIVE-5-1
[  18.4290]ARRIVE-5-4
[  18.5810]ARRIVE-9-5
[  18.6260]ARRIVE-7-2
[  18.7230]ARRIVE-4-1
[  18.7230]OPEN-4-1
[  18.8290]ARRIVE-4-4
[  18.8290]OPEN-4-4
[  18.9230]OUT-69-4-1
[  18.9810]ARRIVE-8-5
[  18.9810]OPEN-8-5
[  19.0300]OUT-4-4-4
[  19.1240]CLOSE-4-1
[  19.1260]ARRIVE-8-2
[  19.1820]IN-60-8-5
[  19.2300]RECEIVE-5-6
[  19.2310]CLOSE-4-4
[  19.3820]CLOSE-8-5
[  19.5240]ARRIVE-5-1
[  19.6260]ARRIVE-9-2
[  19.6270]OPEN-9-2
[  19.6300]ARRIVE-3-6
[  19.7820]ARRIVE-7-5
[  19.8270]OUT-34-9-2
[  19.9240]ARRIVE-6-1
[  19.9240]OPEN-6-1
[  20.0270]CLOSE-9-2
[  20.0300]ARRIVE-2-6
[  20.0300]OPEN-2-6
[  20.1250]IN-2-6-1
[  20.1820]ARRIVE-6-5
[  20.2300]IN-5-2-6
[  20.3060]RECEIVE-9-4
[  20.3250]CLOSE-6-1
[  20.4310]CLOSE-2-6
[  20.5830]ARRIVE-5-5
[  20.7060]ARRIVE-5-4
[  20.7250]ARRIVE-5-1
[  20.8310]ARRIVE-3-6
[  20.9830]ARRIVE-4-5
[  21.1070]ARRIVE-6-4
[  21.1070]OPEN-6-4
[  21.1260]ARRIVE-4-1
[  21.2310]ARRIVE-4-6
[  21.2800]RECEIVE-52-4
[  21.3070]IN-9-6-4
[  21.3830]ARRIVE-3-5
[  21.5070]CLOSE-6-4
[  21.5260]ARRIVE-3-1
[  21.6320]ARRIVE-5-6
[  21.7840]ARRIVE-2-5
[  21.9080]ARRIVE-5-4
[  21.9080]OPEN-5-4
[  21.9270]ARRIVE-2-1
[  21.9270]OPEN-2-1
[  22.0320]ARRIVE-6-6
[  22.1080]OUT-9-5-4
[  22.1080]IN-52-5-4
[  22.1270]OUT-2-2-1
[  22.1840]ARRIVE-1-5
[  22.1840]OPEN-1-5
[  22.3090]CLOSE-5-4
[  22.3280]CLOSE-2-1
[  22.3840]OUT-60-1-5
[  22.4320]ARRIVE-7-6
[  22.5850]CLOSE-1-5
[  22.7090]ARRIVE-4-4
[  22.8330]ARRIVE-8-6
[  23.1090]ARRIVE-3-4
[  23.1100]OPEN-3-4
[  23.2330]ARRIVE-9-6
[  23.3100]OUT-52-3-4
[  23.5100]CLOSE-3-4
[  23.6330]ARRIVE-10-6
[  23.6340]OPEN-10-6
[  23.6820]RECEIVE-48-6
[  23.7800]RECEIVE-49-5
[  23.7800]OPEN-1-5
[  23.8340]OUT-5-10-6
[  23.9810]IN-49-1-5
[  24.0340]CLOSE-10-6
[  24.0350]OPEN-10-6
[  24.1810]CLOSE-1-5
[  24.2350]IN-48-10-6
[  24.4350]CLOSE-10-6
[  24.5800]RECEIVE-55-2
[  24.5810]OPEN-9-2
[  24.5820]ARRIVE-2-5
[  24.7810]IN-55-9-2
[  24.8360]ARRIVE-9-6
[  24.9810]CLOSE-9-2
[  24.9820]ARRIVE-3-5
[  25.2360]ARRIVE-8-6
[  25.2360]OPEN-8-6
[  25.3820]ARRIVE-4-5
[  25.4370]OUT-48-8-6
[  25.4820]ARRIVE-8-2
[  25.4820]OPEN-8-2
[  25.6370]CLOSE-8-6
[  25.6820]OUT-55-8-2
[  25.7830]ARRIVE-5-5
[  25.8820]CLOSE-8-2
[  25.9800]RECEIVE-14-4
[  26.0790]RECEIVE-8-1
[  26.1830]ARRIVE-6-5
[  26.2800]RECEIVE-51-1
[  26.3800]ARRIVE-4-4
[  26.3800]OPEN-4-4
[  26.4800]ARRIVE-3-1
[  26.5810]IN-14-4-4
[  26.5830]ARRIVE-7-5
[  26.6800]RECEIVE-29-3
[  26.7810]CLOSE-4-4
[  26.8800]ARRIVE-4-1
[  26.8800]OPEN-4-1
[  26.9840]ARRIVE-8-5
[  26.9840]OPEN-8-5
[  27.0800]IN-8-4-1
[  27.0800]IN-51-4-1
[  27.1810]ARRIVE-5-4
[  27.1810]OPEN-5-4
[  27.1840]OUT-49-8-5
[  27.2800]ARRIVE-2-3
[  27.2800]OPEN-2-3
[  27.2810]CLOSE-4-1
[  27.3820]OUT-14-5-4
[  27.3840]CLOSE-8-5
[  27.4800]IN-29-2-3
[  27.5820]CLOSE-5-4
[  27.6810]CLOSE-2-3
[  27.6810]ARRIVE-3-1
[  28.0810]ARRIVE-2-1
[  28.2810]RECEIVE-38-1
[  28.2810]ARRIVE-3-3
[  28.4820]ARRIVE-1-1
[  28.4820]OPEN-1-1
[  28.6820]OUT-8-1-1
[  28.6820]OUT-51-1-1
[  28.7800]RECEIVE-31-1
[  28.8810]ARRIVE-4-3
[  28.8820]CLOSE-1-1
[  28.8830]OPEN-1-1
[  29.0830]IN-31-1-1
[  29.2830]CLOSE-1-1
[  29.4820]ARRIVE-5-3
[  29.6830]ARRIVE-2-1
[  29.6840]OPEN-2-1
[  29.7800]RECEIVE-24-1
[  29.8840]IN-38-2-1
[  30.0820]ARRIVE-6-3
[  30.0820]OPEN-6-3
[  30.0840]CLOSE-2-1
[  30.2820]OUT-29-6-3
[  30.4830]CLOSE-6-3
[  30.4840]ARRIVE-3-1
[  30.4850]OPEN-3-1
[  30.6850]OUT-38-3-1
[  30.6850]IN-24-3-1
[  30.8850]CLOSE-3-1
[  31.2850]ARRIVE-4-1
[  31.6860]ARRIVE-5-1
[  31.8800]RECEIVE-23-5
[  31.9800]RECEIVE-39-6
[  32.0800]RECEIVE-64-4
[  32.0860]ARRIVE-6-1
[  32.2800]RECEIVE-7-2
[  32.2800]ARRIVE-9-5
[  32.2810]OPEN-9-5
[  32.3800]ARRIVE-9-6
[  32.3800]OPEN-9-6
[  32.4800]ARRIVE-4-4
[  32.4810]IN-23-9-5
[  32.4860]ARRIVE-7-1
[  32.5800]IN-39-9-6
[  32.6810]CLOSE-9-5
[  32.7800]ARRIVE-7-2
[  32.7800]OPEN-7-2
[  32.7800]CLOSE-9-6
[  32.8800]ARRIVE-3-4
[  32.8800]OPEN-3-4
[  32.8870]ARRIVE-8-1
[  32.9800]IN-7-7-2
[  33.0780]RESET_ACCEPT-3-8-0.3
[  33.0800]IN-64-3-4
[  33.0810]ARRIVE-8-5
[  33.1810]ARRIVE-8-6
[  33.1810]OPEN-8-6
[  33.1860]CLOSE-7-2
[  33.2810]CLOSE-3-4
[  33.2870]ARRIVE-9-1
[  33.3810]OUT-39-8-6
[  33.4800]RECEIVE-68-6
[  33.4810]ARRIVE-7-5
[  33.5810]CLOSE-8-6
[  33.6810]ARRIVE-2-4
[  33.6860]ARRIVE-6-2
[  33.6870]ARRIVE-10-1
[  33.6870]OPEN-10-1
[  33.8820]ARRIVE-6-5
[  33.8870]OUT-31-10-1
[  33.8880]OUT-24-10-1
[  33.9810]ARRIVE-9-6
[  33.9820]OPEN-9-6
[  34.0810]ARRIVE-1-4
[  34.0810]OPEN-1-4
[  34.0880]CLOSE-10-1
[  34.1820]IN-68-9-6
[  34.1860]ARRIVE-5-2
[  34.1860]OPEN-5-2
[  34.2780]RESET_ACCEPT-5-3-0.6
[  34.2810]OUT-64-1-4
[  34.2820]ARRIVE-5-5
[  34.2820]OPEN-5-5
[  34.3820]CLOSE-9-6
[  34.3870]OUT-7-5-2
[  34.4820]CLOSE-1-4
[  34.4820]OUT-23-5-5
[  34.4840]RECEIVE-23-2
[  34.5870]CLOSE-5-2
[  34.5870]OPEN-5-2
[  34.6830]CLOSE-5-5
[  34.6830]RESET_BEGIN-5
[  34.7830]ARRIVE-8-6
[  34.7870]IN-23-5-2
[  34.9870]CLOSE-5-2
[  35.1830]ARRIVE-7-6
[  35.1830]OPEN-7-6
[  35.3830]OUT-68-7-6
[  35.4880]ARRIVE-4-2
[  35.5840]CLOSE-7-6
[  35.8830]RESET_END-5
[  35.9880]ARRIVE-3-2
[  35.9880]OPEN-3-2
[  36.1880]OUT-23-3-2
[  36.2810]RECEIVE-3-6
[  36.3890]CLOSE-3-2
[  36.6800]RECEIVE-61-4
[  36.6800]OPEN-1-4
[  36.6810]ARRIVE-6-6
[  36.6820]OPEN-6-6
[  36.8810]IN-61-1-4
[  36.8820]IN-3-6-6
[  37.0800]RECEIVE-54-2
[  37.0810]CLOSE-1-4
[  37.0820]CLOSE-6-6
[  37.4810]ARRIVE-2-4
[  37.4820]ARRIVE-7-6
[  37.5800]ARRIVE-4-2
[  37.5800]OPEN-4-2
[  37.7800]IN-54-4-2
[  37.8810]ARRIVE-3-4
[  37.8820]ARRIVE-8-6
[  37.9810]CLOSE-4-2
[  37.9820]RECEIVE-27-3
[  37.9820]RESET_BEGIN-3
[  37.9830]RECEIVE-27-1
[  38.1800]RESET_ACCEPT-3-4-0.2
[  38.2820]ARRIVE-4-4
[  38.2830]ARRIVE-9-6
[  38.3830]ARRIVE-9-1
[  38.4810]ARRIVE-5-2
[  38.4810]OPEN-5-2
[  38.6810]OUT-54-5-2
[  38.6820]ARRIVE-5-4
[  38.6830]ARRIVE-10-6
[  38.6830]OPEN-10-6
[  38.7840]ARRIVE-8-1
[  38.8820]CLOSE-5-2
[  38.8830]OUT-3-10-6
[  39.0820]ARRIVE-6-4
[  39.0830]CLOSE-10-6
[  39.1820]RESET_END-3
[  39.1840]ARRIVE-7-1
[  39.1840]OPEN-7-1
[  39.3840]IN-27-7-1
[  39.4820]ARRIVE-7-4
[  39.5850]CLOSE-7-1
[  39.7820]RECEIVE-37-3
[  39.8830]ARRIVE-8-4
[  39.9820]RECEIVE-66-6
[  39.9850]ARRIVE-6-1
[  39.9850]OPEN-6-1
[  40.0820]ARRIVE-7-3
[  40.1840]RECEIVE-41-2
[  40.1850]OPEN-5-2
[  40.1850]OUT-27-6-1
[  40.2830]ARRIVE-9-4
[  40.3820]ARRIVE-9-6
[  40.3820]OPEN-9-6
[  40.3820]ARRIVE-8-3
[  40.3850]IN-41-5-2
[  40.3850]CLOSE-6-1
[  40.5780]RESET_ACCEPT-4-3-0.6
[  40.5800]RECEIVE-42-1
[  40.5820]IN-66-9-6
[  40.5850]CLOSE-5-2
[  40.6820]ARRIVE-9-3
[  40.6820]OPEN-9-3
[  40.6830]ARRIVE-10-4
[  40.6840]OPEN-10-4
[  40.7820]CLOSE-9-6
[  40.8820]IN-37-9-3
[  40.8840]OUT-61-10-4
[  40.8850]RECEIVE-61-6
[  40.9800]ARRIVE-5-1
[  41.0830]CLOSE-9-3
[  41.0840]CLOSE-10-4
[  41.0840]RESET_BEGIN-4
[  41.0850]ARRIVE-4-2
[  41.0850]OPEN-4-2
[  41.1820]ARRIVE-10-6
[  41.1830]OPEN-10-6
[  41.2860]OUT-41-4-2
[  41.3800]ARRIVE-4-1
[  41.3830]RECEIVE-40-2
[  41.3830]ARRIVE-8-3
[  41.3850]OUT-66-10-6
[  41.3850]IN-61-10-6
[  41.4860]CLOSE-4-2
[  41.4860]OPEN-4-2
[  41.5850]CLOSE-10-6
[  41.6830]ARRIVE-7-3
[  41.6860]IN-40-4-2
[  41.7810]ARRIVE-3-1
[  41.7810]OPEN-3-1
[  41.8790]RECEIVE-13-3
[  41.8860]CLOSE-4-2
[  41.9790]RECEIVE-53-5
[  41.9810]IN-42-3-1
[  41.9830]ARRIVE-6-3
[  41.9860]ARRIVE-11-6
[  41.9860]OPEN-11-6
[  42.0790]RECEIVE-50-2
[  42.1810]CLOSE-3-1
[  42.1860]OUT-61-11-6
[  42.2840]ARRIVE-5-3
[  42.2840]RESET_END-4
[  42.3860]CLOSE-11-6
[  42.3870]ARRIVE-5-2
[  42.3870]OPEN-5-2
[  42.5790]ARRIVE-6-5
[  42.5810]ARRIVE-4-1
[  42.5840]ARRIVE-4-3
[  42.5840]OPEN-4-3
[  42.5870]OUT-40-5-2
[  42.6800]RECEIVE-36-1
[  42.7840]OUT-37-4-3
[  42.7870]CLOSE-5-2
[  42.7870]OPEN-5-2
[  42.9820]ARRIVE-5-1
[  42.9840]CLOSE-4-3
[  42.9840]OPEN-4-3
[  42.9870]IN-50-5-2
[  43.0790]RECEIVE-16-2
[  43.0810]RECEIVE-63-6
[  43.1790]ARRIVE-7-5
[  43.1800]OPEN-7-5
[  43.1800]RECEIVE-70-4
[  43.1850]IN-13-4-3
[  43.1870]CLOSE-5-2
[  43.1880]OPEN-5-2
[  43.3800]IN-53-7-5
[  43.3820]ARRIVE-6-1
[  43.3850]CLOSE-4-3
[  43.3880]IN-16-5-2
[  43.4810]ARRIVE-10-6
[  43.5800]CLOSE-7-5
[  43.5880]CLOSE-5-2
[  43.6850]ARRIVE-5-3
[  43.7800]ARRIVE-9-4
[  43.7800]OPEN-9-4
[  43.7820]ARRIVE-7-1
[  43.7820]OPEN-7-1
[  43.8810]ARRIVE-9-6
[  43.9810]IN-70-9-4
[  43.9820]OUT-42-7-1
[  43.9820]IN-36-7-1
[  43.9850]ARRIVE-6-3
[  44.0880]ARRIVE-4-2
[  44.1800]ARRIVE-8-5
[  44.1810]CLOSE-9-4
[  44.1820]CLOSE-7-1
[  44.2820]ARRIVE-8-6
[  44.2850]ARRIVE-7-3
[  44.5830]ARRIVE-8-1
[  44.5860]ARRIVE-8-3
[  44.5880]ARRIVE-3-2
[  44.6820]ARRIVE-7-6
[  44.7800]ARRIVE-9-5
[  44.7810]OPEN-9-5
[  44.7810]ARRIVE-10-4
[  44.7820]OPEN-10-4
[  44.8860]ARRIVE-9-3
[  44.8860]OPEN-9-3
[  44.9810]OUT-53-9-5
[  44.9820]OUT-70-10-4
[  44.9850]ARRIVE-9-1
[  45.0790]RESET_ACCEPT-6-7-0.3
[  45.0820]ARRIVE-6-6
[  45.0820]RESET_BEGIN-6
[  45.0830]RECEIVE-63-3
[  45.0860]OUT-13-9-3
[  45.0890]ARRIVE-2-2
[  45.0890]OPEN-2-2
[  45.1810]CLOSE-9-5
[  45.1820]CLOSE-10-4
[  45.2860]CLOSE-9-3
[  45.2890]OUT-16-2-2
[  45.3850]ARRIVE-10-1
[  45.3860]OPEN-10-1
[  45.4890]CLOSE-2-2
[  45.5860]OUT-36-10-1
[  45.5860]ARRIVE-8-3
[  45.6790]RESET_ACCEPT-4-8-0.3
[  45.7860]CLOSE-10-1
[  45.8870]ARRIVE-7-3
[  45.9890]ARRIVE-1-2
[  45.9890]OPEN-1-2
[  46.1870]ARRIVE-6-3
[  46.1890]OUT-50-1-2
[  46.2820]RESET_END-6
[  46.3900]CLOSE-1-2
[  46.4870]ARRIVE-5-3
[  46.6800]RECEIVE-45-1
[  46.7870]ARRIVE-4-3
[  46.9820]RECEIVE-10-6
[  46.9840]RECEIVE-11-2
[  46.9840]OPEN-1-2
[  47.0800]ARRIVE-11-1
[  47.0800]OPEN-11-1
[  47.0880]ARRIVE-3-3
[  47.1840]IN-11-1-2
[  47.2800]IN-45-11-1
[  47.2830]ARRIVE-7-6
[  47.3840]CLOSE-1-2
[  47.3880]ARRIVE-2-3
[  47.3880]OPEN-2-3
[  47.4800]CLOSE-11-1
[  47.5790]RECEIVE-65-6
[  47.5830]ARRIVE-8-6
[  47.5880]IN-63-2-3
[  47.6800]RECEIVE-28-2
[  47.7880]CLOSE-2-3
[  47.8800]ARRIVE-10-1
[  47.8830]ARRIVE-9-6
[  47.8830]OPEN-9-6
[  47.8850]ARRIVE-2-2
[  48.0830]IN-65-9-6
[  48.0880]ARRIVE-3-3
[  48.2810]ARRIVE-9-1
[  48.2840]CLOSE-9-6
[  48.3850]ARRIVE-3-2
[  48.3850]OPEN-3-2
[  48.3890]ARRIVE-4-3
[  48.5780]RESET_ACCEPT-1-3-0.2
[  48.5840]ARRIVE-8-6
[  48.5840]OPEN-8-6
[  48.5850]OUT-11-3-2
[  48.6810]RECEIVE-46-3
[  48.6810]ARRIVE-8-1
[  48.6810]OPEN-8-1
[  48.6830]RECEIVE-43-2
[  48.6890]ARRIVE-5-3
[  48.6920]RESET_BEGIN-4
[  48.7840]IN-10-8-6
[  48.7850]CLOSE-3-2
[  48.7850]OPEN-3-2
[  48.8820]OUT-45-8-1
[  48.9840]CLOSE-8-6
[  48.9860]IN-28-3-2
[  48.9890]ARRIVE-6-3
[  49.0820]CLOSE-8-1
[  49.0820]RESET_BEGIN-1
[  49.1860]CLOSE-3-2
[  49.2850]ARRIVE-7-6
[  49.2890]ARRIVE-7-3
[  49.2890]OPEN-7-3
[  49.4890]OUT-63-7-3
[  49.5850]ARRIVE-6-6
[  49.5850]OPEN-6-6
[  49.6860]ARRIVE-2-2
[  49.6860]OPEN-2-2
[  49.6900]CLOSE-7-3
[  49.7850]OUT-10-6-6
[  49.8860]OUT-28-2-2
[  49.8870]IN-43-2-2
[  49.8920]RESET_END-4
[  49.9850]CLOSE-6-6
[  49.9900]ARRIVE-6-3
[  50.0870]CLOSE-2-2
[  50.2820]RESET_END-1
[  50.2860]ARRIVE-5-6
[  50.2900]ARRIVE-5-3
[  50.5860]ARRIVE-4-6
[  50.5870]ARRIVE-1-2
[  50.5870]OPEN-1-2
[  50.5900]ARRIVE-4-3
[  50.7870]OUT-43-1-2
[  50.8860]ARRIVE-3-6
[  50.8910]ARRIVE-3-3
[  50.8910]OPEN-3-3
[  50.9880]CLOSE-1-2
[  51.0910]IN-46-3-3
[  51.1860]ARRIVE-2-6
[  51.1870]OPEN-2-6
[  51.2910]CLOSE-3-3
[  51.3870]OUT-65-2-6
[  51.5870]CLOSE-2-6
[  51.5910]ARRIVE-4-3
[  51.8910]ARRIVE-5-3
[  52.1920]ARRIVE-6-3
[  52.4920]ARRIVE-7-3
[  52.7920]ARRIVE-8-3
[  53.0920]ARRIVE-9-3
[  53.0920]OPEN-9-3
[  53.2930]OUT-46-9-3
[  53.4930]CLOSE-9-3
